ctk/idp/src/main/kotlin/org/codice/compliance/web/slo/PostSLOTest.kt

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.compliance.web.slo
+
+import com.jayway.restassured.RestAssured
+import de.jupf.staticlog.Log
+import io.kotlintest.specs.StringSpec
+import org.codice.compliance.debugWithSupplier
+import org.codice.compliance.utils.TestCommon.Companion.EXAMPLE_RELAY_STATE
+import org.codice.compliance.utils.TestCommon.Companion.createDefaultLogoutRequest
+import org.codice.compliance.utils.TestCommon.Companion.loginAndGetCookies
+import org.codice.compliance.utils.TestCommon.Companion.sendPostLogoutRequest
+import org.codice.compliance.utils.TestCommon.Companion.signAndEncodeToString
+import org.codice.compliance.utils.getBindingVerifier
+import org.codice.compliance.verification.core.responses.CoreLogoutResponseProtocolVerifier
+import org.codice.security.saml.SamlProtocol.Binding.HTTP_POST
+
+class PostSLOTest : StringSpec() {
+
+    init {
+        RestAssured.useRelaxedHTTPSValidation()
```

Comment:
I wasn't sure what this did so I looked it up. It configures RestAssured to accept invalid certificates.
https://static.javadoc.io/com.jayway.restassured/rest-assured/2.8.0/com/jayway/restassured/RestAssured.html#useRelaxedHTTPSValidation--